examples/nodejs/src/experimental-features/applicable-inputs.ts

```diff
@@ -0,0 +1,512 @@
+/**
+This is an experimental module that aims to replace the current `next` features.
+TODO: After PR-8891 and as part of PLT-9054 move these to the runtime-lifecycle package
+      and create a github discussion to modify the backend.
+ */
+
+import {
+  MarloweState,
+  Party,
+  Contract,
+  Deposit,
+  Choice,
+  Input,
+  ChosenNum,
+  Environment,
+  Timeout,
+  getNextTimeout,
+  datetoTimeout,
+  Case,
+  Action,
+  Notify,
+  InputContent,
+  RoleName,
+  Token,
+  Bound,
+} from "@marlowe.io/language-core-v1";
+import {
+  applyAllInputs,
+  convertReduceWarning,
+  evalObservation,
+  evalValue,
+  inBounds,
+  Payment,
+  reduceContractUntilQuiescent,
+  TransactionWarning,
+} from "@marlowe.io/language-core-v1/semantics";
+import { AddressBech32, ContractId, PolicyId } from "@marlowe.io/runtime-core";
+import { RestClient } from "@marlowe.io/runtime-rest-client";
+import { WalletAPI } from "@marlowe.io/wallet";
+import * as Big from "@marlowe.io/adapter/bigint";
+import { Monoid } from "fp-ts/lib/Monoid.js";
+import * as R from "fp-ts/lib/Record.js";
+
+type ActionApplicant = Party | "anybody";
+
+export interface AppliedActionResult {
+  /**
+   * What inputs needs to be provided to apply the action
+   */
+  inputs: Input[];
+
+  /**
+   * What is the environment to apply the inputs
+   */
+  environment: Environment;
+  /**
+   * What is the new state after applying an action and reducing until quiescent
+   */
+  reducedState: MarloweState;
+  /**
+   * What is the new contract after applying an action and reducing until quiescent
+   */
+  reducedContract: Contract;
+  /**
+   * What warnings were produced while applying an action
+   */
+  warnings: TransactionWarning[];
+  /**
+   * What payments were produced while applying an action
+   */
+  payments: Payment[];
+}
+
+interface CanNotify {
+  type: "Notify";
+  policyId: PolicyId;
```

Comment:
what `policyId` are we talking about here ? I don't understand

Reply:
The policyId of a created contract. I need this for filtering my actions. I need to hide this away somehow